x-pack/plugins/security_solution/common/search_strategy/security_solution/hosts/details/index.ts

```diff
@@ -22,3 +22,41 @@ export interface HostDetailsRequestOptions extends Partial<RequestOptionsPaginat
   timerange: TimerangeInput;
   inspect?: Maybe<Inspect>;
 }
+
+export interface AggregationRequest {
+  [aggField: string]: {
+    terms?: {
+      field?: string;
+      missing?: string;
+      size?: number;
+      script?: {
+        source: string;
+        lang: string;
+      };
+      order?: {
+        [aggSortField: string]: Direction;
+      };
+    };
+    max?: {
+      field: string;
+    };
+    aggs?: {
+      [aggSortField: string]: {
```

Comment:
are you sure that 's `aggSortField` here?

Reply:
yeah, I took it from https://github.com/elastic/kibana/blob/f21417660af7dfc65bea05bf85253bb96c837c87/x-pack/plugins/security_solution/server/lib/types.ts#L177

Reply:
maybe we could switch to update elasticsearch types now? https://github.com/elastic/kibana/pull/83808

Reply:
ok, replaced line 27 - 62 with estypes.AggregationContainer